.github/workflows/api-e2e.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Run Admin API
```

Comment:
@lchavarriaga 
CopyToDockerContext, RemoveDockerContextFiles functions should be called from here to copy over the required files before docker compose.

Reply:
@lchavarriaga 
I added the steps on api-e2e.yml to copy those required application folder and NuGet.Config file to Docker/ Application folder.